packages/go/cypher/backend/pgsql/format_test.go

```diff
@@ -0,0 +1,714 @@
+		{
+			ID:      301,
+			Source:  "match (s) where s.name = '123' match (e) where e.name = 'lol' return s.name, e",
+			Ignored: true,
+		},
+	}
+}
+
+func TestPGSQLEmitter(t *testing.T) {
+	var (
+		runnable     []TestCase
+		exclusiveRun bool
+	)
+
+	for _, testCase := range Suite() {
+		if testCase.Ignored {
+			continue
+		}
+
+		if testCase.Exclusive {
+			if !exclusiveRun {
+				runnable = runnable[:0]
+				exclusiveRun = true
+			}
+
+			runnable = append(runnable, testCase)
+		} else if !exclusiveRun {
+			runnable = append(runnable, testCase)
+		}
+	}
+
+	for _, testCase := range runnable {
+		var regularQuery *model.RegularQuery
+
+		if testCase.Query != nil {
+			builtQuery, err := testCase.Query.Build()
+			require.Nilf(t, err, "test case %d: %v", testCase.ID, err)
+
+			regularQuery = builtQuery
+
+		} else {
+			parsedQuery, parseErr := frontend.ParseCypher(frontend.NewContext(), testCase.Source)
+			require.Nilf(t, parseErr, "test case %d: %v", testCase.ID, parseErr)
+
+			regularQuery = parsedQuery
+		}
+
+		var (
+			buffer     = &bytes.Buffer{}
+			kindMapper = KindMapper{
+				known: map[string]int16{
+					"NodeKindA": 1,
+					"NodeKindB": 2,
+					"NodeKindC": 3,
+					"EdgeKindA": 100,
+					"EdgeKindB": 101,
+					"EdgeKindC": 102,
+				},
+			}
+
+			parameters, translationErr = pgsql.Translate(regularQuery, kindMapper)
+		)
+
+		if testCase.Error {
+			if translationErr != nil {
+				continue
+			}
+
+			var (
+				emitter    = pgsql.NewEmitter(false, kindMapper)
+				emitterErr = emitter.Write(regularQuery, buffer)
+			)
+
+			require.NotNilf(t, emitterErr, "test case %d: %v", testCase.ID, emitterErr)
+		} else {
+			require.Nilf(t, translationErr, "test case %d: %v", testCase.ID, translationErr)
+
+			if testCase.ExpectedParameters != nil {
+				require.Equal(t, testCase.ExpectedParameters, parameters)
+			}
+
+			var (
+				emitter    = pgsql.NewEmitter(false, kindMapper)
+				emitterErr = emitter.Write(regularQuery, buffer)
+			)
+
+			require.Nilf(t, emitterErr, "test case %d: %v", testCase.ID, emitterErr)
+			require.Equalf(t, testCase.Expected, buffer.String(), "test case %d", testCase.ID)
+		}
+	}
+}
+
+func TestBinder(t *testing.T) {
+	var (
+		binder                 = pgsql.NewBinder()
+		regularQuery, parseErr = frontend.ParseCypher(frontend.DefaultCypherContext(), "match (s) with s as m return s")
+		binderErr              = binder.Scan(regularQuery)
+	)
+
+	require.Nil(t, parseErr)
+	require.Nil(t, binderErr)
+
+	require.True(t, binder.IsBound("s"))
+	require.True(t, binder.IsPatternBinding("s"))
+	require.True(t, binder.IsBound("m"))
+
+	// TODO: This might want to be true depending on how references play out during joins
+	require.False(t, binder.IsPatternBinding("m"))
+}
+
+//func TestCypherEmitter_HappyPath(t *testing.T) {
+//	test.LoadFixture(t, test.PositiveTestCases).Run(t)
+//}
+//
+//func TestCypherEmitter_NegativeCases(t *testing.T) {
+//	test.LoadFixture(t, test.NegativeTestCases).Run(t)
+//}
```

Comment:
please remove dead code or comment as to why it's commented out

Reply:
Patched up - this was me copying the format test for cypher and not removing the dead testing functions